tests/consensus_tests/test_shard_transfer_rate_limiting.py

```diff
@@ -82,13 +82,13 @@ def test_shard_transfer_rate_limiting(tmp_path: pathlib.Path):
     shard_id = collection_cluster_info["local_shards"][0]["shard_id"]
     source_peer_id = collection_cluster_info["peer_id"]
 
-    # set rate limiting
+    # Set rate limiting
     r = requests.patch(
         f"{source_uri}/collections/test_collection", json={
             "strict_mode_config": {
                 "enabled": True,
                 "write_rate_limit": 1,
-                # TODO(ratelimit) validate read limits as well "read_rate_limit": 1
+                "read_rate_limit": 1,
```

Comment:
without the fix, enabling this line makes the test fail due to read rate limit reached :)